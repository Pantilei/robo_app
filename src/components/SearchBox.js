import React from "react";

const SearchBox = ({ onSearchChange, inputValue }) => {
  return (
    <div className="pa2">
      <input
        aria-label="Search Robots"
        className="pa3 ba b--green bg-lightest-blue"
        type="search"
        placeholder="search for robot"
        onChange={onSearchChange}
        value={inputValue}
      />
    </div>
  );
};

export default SearchBox;
