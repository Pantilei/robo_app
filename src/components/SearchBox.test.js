import React from "react";
import { shallow } from "enzyme";
import SearchBox from "./SearchBox";

it("Testing SearchBox component", () => {
  expect(shallow(<SearchBox />)).toMatchSnapshot();
});
