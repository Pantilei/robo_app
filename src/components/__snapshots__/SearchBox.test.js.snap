// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing SearchBox component 1`] = `ShallowWrapper {}`;
